packages/clay-data-provider/stories/index.tsx

```diff
@@ -24,7 +24,7 @@ const ClayDataProviderWithVariablesAndStorage = () => {
 		<div className="col-md-4">
 			<ClayDataProvider
 				fetchPolicy={FetchPolicy.CacheFirst}
-				link="https://rickandmortyapi.com/api/character"
+				link="https://rickandmortyapi.com/api/character/"
```

Comment:
How come you added this here?

Reply:
Ah yeah, that was because it was causing a 302 (Temporary redirect to the address with `/`), I added it to not see it anymore on the tab network 😁.